Automattic/pocket-casts-ios
# updated privacy icon

Fixes #644 

Please include a summary of the changes and the related issue.

## To test
Go to settings screen and see new icon

![Simulator Screen Shot - iPhone 14 Pro - 2023-01-13 at 16 51 29](https://user-images.githubusercontent.com/1335657/212442749-ceb9e716-504f-467d-8104-a780d68d602c.png)
